eya_def_tools/eya_def_tools/data_models/plant_performance.py

```diff
@@ -84,11 +104,11 @@ class PlantPerformanceSubcategoryElement(EyaDefBaseModel):
             "subcategory element."
         ),
     )
-    is_independent: bool = pdt.Field(
+    is_statistically_independent: bool = pdt.Field(
```

Comment:
Looking at this now I thought `is_statistically_independent` is clearer than `is_independent`, since the word "independent" could also mean other things (e.g. the independence of the party undertaking the assessment).